src/pages/en/guides/content-collections.mdx

```diff
@@ -16,166 +16,196 @@ import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro
   <Since v="2.0.0" />
 </p>
 
-Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. Collections may be helpful if you:
+**Content collections** are the best way to work with Markdown and MDX in any Astro project. Content collections are a feature of Astro that help manage your content files in a project. Collections help to organize your content, validate your frontmatter, and provide automatic TypeScript type-safety for all of your content.
 
-- **Plan to use Markdown content in multiple areas** of your site (landing pages, footers, navigation, etc).
-- **Want Astro to enforce frontmatter fields,** and fail if fields are missing (e.g. every blog post should have a title and description).
 
-## Getting started
+## What are Content Collections?
 
-First, run the [`astro dev`](/en/reference/cli-reference/#astro-dev), [`astro build`](/en/reference/cli-reference/#astro-build), or [`astro sync`](/en/reference/cli-reference/#astro-sync) commands to generate types. This will make the `astro:content` module available for [querying](#querying-content-collections) and [configuring](#configuring-collections) your content collections.
+A **content collection** is any directory inside the reserved `src/content` project directory, such as `src/content/newsletter` and `src/content/blog`. Only content collections are allowed inside the `src/content` directory. This directory cannot be used for anything else.
 
-### Update TypeScript configuration
+A **content entry** is any piece of content stored inside of your content collection. Content entries are stored as either Markdown (`.md`) or MDX (`.mdx`) files, saved inside of content collection directory. Your content entries can be named anything you want, but we recommend using a consistent naming scheme to make it easier to find and organize your content. 
```

Comment:
```suggestion
A **content entry** is any piece of content stored inside of your content collection. Content entries are stored as either Markdown (`.md`) or MDX (`.mdx`) files, saved inside of the content collection directory. Your content entries can be named anything you want, but we recommend using a consistent naming scheme to make it easier to find and organize your content. 
```
At a minimum, there's a missing word, so that's what I've corrected.

Additionally: I feel like this got wordier.  :smile:

I would personally hesitate to put the word "any" in the reader's mind when we're about to restrict usage to Markdown and MDX files. This would be my suggestion:

> A **content entry** is a Markdown (`.md`) or MDX (`.mdx`) file stored inside of the content collection directory. 

I also don't feel like the advice on naming is particularly helpful? Are there *absolutely no* restrictions on file names? (And if so, I'd phrase it that way because that could be helpful info worth noting, and a little more in keeping with the docs tone.) It can start with a special character? An underscore? Contain emojis? I also don't think it's needed to suggest consistent naming, but if phrased in terms of letting them know that it's a "flat structure" or something like that, maybe?

Reply:
This was something we went back and forth on last week, but settled on the implemented behavior that filenames can be literally anything! I'd still rather be much more strict personally (only lower-case, dashes, etc) but this felt too late to change. 

Post-launch, I would love for us to pick this back up as a docs "recommendation", so that even if we keep supporting anything, we could add a recommendation here for a certain format.